### ISSUE TITLE
[Startup errors]

### ISSUE THREAD

**Issue by xfurry** [cmangos/wotlk-db#37]
Poke @Grz3s about some core startup errors:
I'm using latest DB version [615]

```
2016-12-10 12:02:12 ERROR:Item (Entry: 5513) has redundant real-time duration flag in ExtraFlags, item not have duration
2016-12-10 12:02:12 ERROR:Item (Entry: 5514) has redundant real-time duration flag in ExtraFlags, item not have duration
2016-12-10 12:02:12 ERROR:Item (Entry: 8007) has redundant real-time duration flag in ExtraFlags, item not have duration
2016-12-10 12:02:12 ERROR:Item (Entry: 8008) has redundant real-time duration flag in ExtraFlags, item not have duration
2016-12-10 12:02:12 ERROR:Item (Entry: 22044) has redundant real-time duration flag in ExtraFlags, item not have duration
2016-12-10 12:02:12 ERROR:Item (Entry: 33312) has redundant real-time duration flag in ExtraFlags, item not have duration
```

**Comment by Grz3s** [comment]
@xfurry im 99% thats Core issue .. and CoreTeam knows about it already (@cyberium)

**Comment by killerwife** [comment]
@namreeb is the one who made the core patch that introduced this

**Comment by Darth-Postal** [comment]
I did some research on this. 

In https://github.com/cmangos/mangos-wotlk/commit/e32b9c250b7d22a276e0c01c5958b497b6a27448 flag ITEM_EXTRA_NON_CONSUMABLE was removed.

Before this commit, we had 3 cases for ExtraFlags:
0 - no flags.
1 - Non-consumable flag. The item will not be deleted if it has no charges left. **This is the flag that Mana Gems used to have.**
2 - Realtime duration flag. Item's duration will count down even if player is offline.

Part of 12944 was https://github.com/cmangos/mangos-wotlk/blob/e32b9c250b7d22a276e0c01c5958b497b6a27448/sql/updates/mangos/12944_01_mangos_item_extraflags.sql
```sql
ALTER TABLE db_version CHANGE COLUMN required_12943_01_mangos_flametongue required_12944_01_mangos_item_extraflags bit;

UPDATE item_template SET extraFlags = 1 WHERE extraFlags != 0;
```

This changed OLD ExtraFlag=1 (Non-consumable) to NEW ExtraFlag=1 (Realtime duration).

This is wrong. The query **should** have been:
```sql
ALTER TABLE db_version CHANGE COLUMN required_12943_01_mangos_flametongue required_12944_01_mangos_item_extraflags bit;

UPDATE item_template SET spellcharges_1 = ABS (spellcharges_1), spellcharges_2 = ABS (spellcharges_2), spellcharges_3 = ABS (spellcharges_3), spellcharges_4 = ABS (spellcharges_4), spellcharges_5 = ABS (spellcharges_5), extraFlags = 0 WHERE extraFlags = 1;
UPDATE item_template SET extraFlags = 1 WHERE extraFlags = 2;
```

Fortunately, the only items that used to have Non-Consumable flag were the Mana Gems. I actually checked the pre-12944 version of DB. So we can just update these 6 items to new standard and we're good to go.

_______________________________________________________________________________
TL/DR: Both start-up errors and now-missing Non-consumable flag can be fixed with:
```sql
UPDATE item_template SET spellcharges_1 = ABS (spellcharges_1), spellcharges_2 = ABS (spellcharges_2), spellcharges_3 = ABS (spellcharges_3), spellcharges_4 = ABS (spellcharges_4), spellcharges_5 = ABS (spellcharges_5), extraFlags = 0 WHERE entry IN (5513, 5514, 8007, 8008, 22044, 33312);
```

**Comment by Grz3s** [comment]
@namreeb
